pkgs/tools/admin/aws-vault/default.nix

```diff
@@ -32,6 +39,12 @@ buildGoModule rec {
     -X main.Version=v${version}
   '';
 
+  doInstallCheck = true;
+
+  installCheckPhase = ''
+    $out/bin/aws-vault --version 2>&1 | grep ${version} > /dev/null
```

Comment:
Does this test that `xdg-utils` is being found?

An even better version would be to add `PATH=/not/exist ` before the command to make sure it doesn't source anything from the PATH.

Reply:
Hmm, it doesn't actually test `xdg-utils`, just a simple sanity check to make sure that version is printed and binary is executable 🙂 . It's not necessary though.
The `xdg-utils` dependency is [implicit](https://github.com/99designs/aws-vault/blob/0c10c45693116608fb3765be304f554189f0df01/cli/login.go#L137). To properly test it we would need to have credentials 🤷🏻